doc/manual/src/release-notes/rl-next.md

```diff
@@ -64,3 +64,5 @@
     ```
 
   This makes it match `nix derivation show`, which also maps store paths to information.
+
+- `XDG_DATA_DIRS` is now populated properly, so completion scripts, .desktop files and similar data installed via Nix can be detected properly by programs not installed via Nix.
```

Comment:
We should be clear on the boundary conditions. This is not magic and not revolutionary. At this point I still wouldn't recommend anyone to use the imperative package management features. I think this strikes the right, factual tone:

```suggestion
- When Nix is installed using the [binary installer], in supported shells (Bash, Zsh, Fish) [`XDG_DATA_DIRS`] is now populated with the path to the `/share` subdirectory of the current profile.
  This means that command completion scripts, `.desktop` files, and similar artifacts installed via [`nix-env`] or [`nix profile`] (experimental) can be found by any program that follows the [XDG Base Directory Specification].
```

Please add the links, I don't have a keyboard and large screen right now.

Reply:
Good point and well worded! I added the links, built the docs and verified all of them were indeed linking to the correct locations :+1: